zazu/repo/commands.py

```diff
@@ -55,22 +55,27 @@ def ci(ctx):
 
 
 @repo.command()
-@click.argument('repository')
+@click.argument('repository_url')
 @click.option('--nohooks', is_flag=True)
 @click.pass_context
-def clone(ctx, repository, nohooks):
-    """Clone and initialize a repo"""
+def clone(ctx, repository_url, nohooks):
+    """Clone and initialize a repo
+
+    Args:
+        repository_url(str):url of the repository to clone
+
+    flags:
+        --nohooks: does not install git hooks in the cloned repo
+    """
     try:
-        Git().clone(repository)
+        git.Git().clone(repository_url)
         click.echo('Repository successfully cloned')
 
         if not nohooks:
             click.echo('Installing Git Hooks')
-            zazu.git_helper.install_git_hooks(repository.rsplit('/', 1)[-1].replace('.git', ''))
-            click.echo('Done')
-
-    except GitCommandError:
-        click.echo('Error cloning repository: {}'.format(repository))
+            zazu.git_helper.install_git_hooks(repository_url.rsplit('/', 1)[-1].replace('.git', ''))
+    except git.GitCommandError, err:
+        sys.stderr.write('Error: {}{}'.format(str(err), '\n'))
```

Comment:
This also isn't done in the rest of zazu. I think we should `raise ClickException(err)` in this case that will spit out the error.